pulumi/aws/README.md

```diff
@@ -146,15 +145,40 @@ deployment.
 ### Prometheus
 
 Prometheus is deployed and configured to enable the collection of metrics for all components that have
-properties `prometheus.io:scrape: true` set in the annotations
-(along with any other connection information). This includes the prometheus `node-exporter`
-daemonset which is deployed in this step as well.
+a defined service monitor. At installation time, the deployment will instantiate:
+- Node Exporters
+- Kubernetes Service Monitors
+- Grafana preloaded with dashboards and datasources for Kubernetes management
+- The NGINX Ingress Controller
+- Statsd receiver
+
+The former behavior of using the `prometheus.io:scrape: true` property set in the annotations
+indicating pods where metrics should be scraped has been deprecated, and these annotations will
+be removed in the near future.
+
+Also, the standalone Grafana deployment has been removed from the standard deployment scripts, but has been left as 
+a project in the event someone wishes to run this standalone.
+
+Finally, this namespace will hold service monitors created by other projects, for example the Bank of Sirius
+deployment currently deploys a service monitor for each of the postgres monitors that are deployed.
+
+Notes: 
+1. The NGINX IC needs to be configured to expose prometheus metrics; this is currently done by default.
+2. The default address binding of the `kube-proxy` component is set to `127.0.0.1` and as such will cause errors when the 
+canned prometheus scrape configurations are run. The fix is to set this address to `0.0.0.0`. An example manifest
+has been provided in [prometheus/extras](./prometheus/extras) that can be applied against your installation with 
+`kubectl apply -f ./filename`. Please only apply this change once you have verified that it will work with your 
+version of Kubernetes.
+3. The _grafana_ namespace has been maintained in the conifugration file to be used by the prometheus operator deployed
+version of Grafana. This version only accepts a password; you can still specify a username for the admin account but it 
+will be silently ignored.
 
-This also pulls data from the NGINX KIC, provided the KIC is configured to allow prometheus access (which is enabled by
-default).
 
 ### Grafana
 
+**NOTE:** This deployment has been deprecated but the project has been left as an example on how to deploy Grafana in this 
```

Comment:
Let's just delete and point folks to the git history. We don't want to carry this forward. Thoughts?

Reply:
I went back and forth on this. Part of me wanted to delete it, but then another part started down the "well, what if the user wants to swap out prometheus for something else and still wants grafana?"

If we go to a modular approach where the user runs a script and answers prompts as to what they want / don't want, I feel that just keeping it in place (preferably with a few tests around it to make sure it works) would be fine - since I'm pulling from the mainline grafana builds, we could just manage it like the other dependencies.

That said, I'm not married to this idea - so let me know what you think in light of that.

Reply:
I say, let's delete it. It will always be in the source history and we can always come back and add it again after we have better support for multiple options.

Reply:
Deleted in last commit.